src/demos/calcite-block.html

```diff
@@ -154,6 +154,18 @@ <h2>Header + collapsible + control = no collapsible icon</h2>
           <div>Some content</div>
         </calcite-block>
 
+        <h2>Disabled</h2>
+
+        <calcite-block disabled heading="this block is disabled" collapsible open>
+          <div>Some content</div>
+        </calcite-block>
+
+        <h2>Loading</h2>
+
+        <calcite-block loading heading="this block is loading" collapsible open>
+          <div>Some content</div>
+        </calcite-block>
+
```

Comment:
I don't think you need these since @asangma added the toggle buttons at the top, which will toggle all blocks to be disabled/loading.

Reply:
You can tell I haven't used those toggles. 😄Will remove!

Reply:
Indeed. @jcfranco Do you think we need these since we have the buttons at the top of the demo?